apps/android/README.md

```diff
@@ -23,6 +23,25 @@ yarn build
 yarn android
 ```
 
+## Using an Apple Silicon Mac (As of July 2021)
+
+- The virtual devices in the AVD Manager with arm64 images still have some bugs. To get the current best emulator, download it from Github: [version 3](https://github.com/google/android-emulator-m1-preview).
+
+- Since the emulator is an emulator, you need open it manually first. Then open Android Studio from `/apps/android/src`, choose 'emulator-5554' as the device and run the app.
+- To connect to metro run the following and reload the app:
+
+```sh
+adb reverse tcp:8081 tcp:8081
+yarn start
+```
+
+- APKs: The emulator is running on arm64 so any APKs that need to be installed need to be for arm64. Here is a [link](https://www.apkmirror.com/apk/google-inc/android-accessibility-suite/android-accessibility-suite-9-1-0-358315219-release/android-accessibility-suite-9-1-0-358315219-android-apk-download/) to the accessibility APK which includes TalkBack (voiceover)
+  - The ADB location is not automatically detected on the emulator so you need to manually set it.
+    1.  Go to expanded settings (three dots on the side) -> settings
+    2.  Turn off "Use detected ADB location"
+    3.  Click the folder and find the location of adb (~/Library/Android/sdk/platform-tools/adb). You will need to show hidden folders (Shift + CMD + .) to see Library
```

Comment:
it seems bit inconsistent some we talk about "cd foo" on the terminal sometime we talk about finder guy stuff. should we just unify and just do all terminal commands? also, "ls -l" also shows hidden files in terminal if you need the equivalent.

Reply:
This specifically is within the Android emulator settings panel so terminal commands won't work. Also for this instance, the field won't accept a copy/pasted file location so you're required to go through finder to find the location.